domain/schema/model/sql/0003-password.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE password_hash_algorithm (
+    id INT PRIMARY KEY,
```

Comment:
Missed a `NOT NULL` here.

Reply:
Indeed

Reply:
No, int and integer doesn't need not null, only text does.